src/components/SecondaryPanes/Frames/Group.js

```diff
@@ -155,3 +156,4 @@ export default class Group extends Component<Props, State> {
 }
 
 Group.displayName = "Group";
+Group.contextTypes = contextTypes;
```

Comment:
does group need contextTypes?

Reply:
not for now, but soon there should be a localized tooltip there (see https://github.com/devtools-html/debugger.html/issues/7209)

Reply:
lets add it in #7209 then.